From 56ca96430ba9b8798a28e788a66956b5d4974d52 Mon Sep 17 00:00:00 2001
From: Mario Carrillo <mario.alfredo.c.arevalo@intel.com>
Date: Fri, 19 Feb 2016 17:40:45 -0600
Subject: [PATCH] Add name correction to dpdk combine lib

When OpenvSwitch is built with DPDK libraries,
it links to a single combined library called
libdpdk.so however the configurartion file had
a different name.

Signed-off-by: Mario Carrillo <mario.alfredo.c.arevalo@intel.com>
---
 Makefile.am  | 2 +-
 acinclude.m4 | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 59a1466..abb4945 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4,7 +4,7 @@
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without warranty of any kind.
-
+#
 AUTOMAKE_OPTIONS = foreign subdir-objects
 ACLOCAL_AMFLAGS = -I m4
 SUBDIRS = datapath
diff --git a/acinclude.m4 b/acinclude.m4
index 45cfaf6..90bb708 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -172,7 +172,7 @@ AC_DEFUN([OVS_CHECK_DPDK], [
 
     DPDK_INCLUDE=$RTE_SDK/include
     DPDK_LIB_DIR=$RTE_SDK/lib
-    DPDK_LIB="-lintel_dpdk"
+    DPDK_LIB="-ldpdk"
     DPDK_EXTRA_LIB=""
 
     AC_COMPILE_IFELSE(
-- 
2.5.0

